packages/bff/.env

```diff
@@ -40,6 +40,19 @@ SELFCARE_V2_URL=localhost
 SELFCARE_V2_API_KEY=key
 EXPORT_ESERVICE_CONTAINER="interop-application-import-export-local"
 EXPORT_ESERVICE_PATH="local/eservices-export"
+IMPORT_ESERVICE_CONTAINER="interop-application-import-export-local"
+IMPORT_ESERVICE_PATH="local/eservices-export"
 PRESIGNED_URL_GET_DURATION_MINUTES=5000
 
 RISK_ANALYSIS_DOCUMENTS_PATH="risk-analysis/docs"
+
+S3_BUCKET=interop-local-bucket
```

Comment:
I know it's probably always the same container but in Scala there are different vars for all the different containers: one for eservice documents, one for agreement consumer documents, etc. so that they can be distinguished in the config if needed.

I would remove this `S3_BUCKET` var and use `ESERVICE_DOCUMENTS_CONTAINER`

<img width="947" alt="image" src="https://github.com/user-attachments/assets/f784df7d-a1ee-4ab3-a8da-2d0e48581b49">

Reply:
Hi @ecamellini, I'm already using the `IMPORT_ESERVICE_CONTAINER`, but I need to remove `S3_BUCKET` here since it is duplicated

Reply:
I'm also talking about this other usage: https://github.com/pagopa/interop-be-monorepo/pull/940/files#diff-ff2d451d4a77d58ba3f4b6773b89f859aa3846d92e4c497915741cc6cf8f652dR217

There, you're using `S3_BUCKET`, but Scala uses `ESERVICE_DOCUMENTS_CONTAINER` which is not the same as `IMPORT_ESERVICE_CONTAINER`

Reply:
In general, any usage related to eservice documents should probably use ESERVICE_DOCUMENTS_CONTAINER instead of S3_BUCKETS